### PR TITLE
completed issue22 generating wallet address and private key

### PR DESCRIPTION
**Pull request by kdbusiness121**
issue #22 is completed
1. generating Wallet Address
2. generating Private key

### CODE REVIEW

**Comment by gedanziger** [comment]
@kdbusiness this is with bitg SDK? It looks like standard BTC prefix. I will ask the SDK engineer about an update.

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "ajv": "^6.5.0",
     "antd": "^3.4.4",
     "axios": "^0.18.0",
+    "bitcoinjs-lib": "^4.0.2",
```
Comment (0xweb3builder) [inline package.json:9]: this module should be updated when bitcoingreenjs-lib is published.

Comment (kdbusiness121) [reply package.json:9]: Yes

**Review by gedanziger — APPROVED** [review]

**Comment by 0xweb3builder** [comment]
we will update lib package when bitcoingreenjs-lib published.

**Pull request merged**